src/crepe/Particle.cpp

```diff
@@ -2,8 +2,8 @@
 
 using namespace crepe;
 
-void Particle::reset(uint32_t lifespan, const vec2 & position, const vec2 & velocity,
-					 double angle) {
+void Particle::reset(unsigned int lifespan, const vec2 & position, const vec2 & velocity,
```

Comment:
Omdat `delta_time` overal nu uitgedrukt is in `duration_t` zou ik verwachten dat die ook hier gebruikt zou kunnen worden. Dat maakt de berekeningen overal makkelijker omdat je niet meer de `std::chrono::duration` om hoeft te zetten naar een float/double

Reply:
Niels wou niet dat overal chrono geinclude moest worden alleen voor dit.

Wat bedoel je met 'maakt de berekeningen makkelijker'? 
bedoel je dat dit tijd kost?
`float dt = loop_timer.get_scaled_fixed_delta_time().count();`

Reply:
Daarmee bedoel ik dat je niet om hoeft te rekenen van `std::chrono::duration` naar `float`, en daarmee ook niet hoeft te gokken/documenteren welke grootheid die float is. Zoals ik het begrijp moet er wel omgerekend worden als je van `chrono::duration` naar `float` wil omdat de standaard tijdresolutie die we gebruiken op microseconden staat. Je maakt al gebruik van `manager/LoopTimerManager.h` dus je hoeft zelf `<chrono>` niet te includen omdat die daar al geinclude is voor de typedef `duration_t`.

Reply:
Het staat er gedocumenteerd bij ik vind dat voldoende.
Ik heb enkele redenen maar alle andere system doen het nu ook zo.

Reply:
Als je dit wilt kunnen we het in de meeting overleggen. Maar dat moet dit in alle systemen wel gelijk blijven.